wallet/src/de/schildbach/wallet/ui/main/MainViewModel.kt

```diff
@@ -134,6 +142,10 @@ class MainViewModel @Inject constructor(
     val isNetworkUnavailable: LiveData<Boolean>
         get() = _isNetworkUnavailable
 
+    private val _stakingAPY = MutableLiveData<Double>()
+    val stakingAPY: LiveData<Double>
+        get() = _stakingAPY
```

Comment:
Explore fragment is hosted in the `MainActivity` in DashPay, so staking API is moved here. It probably makes more sense too, since CrowdNode APY doesn't relate to Explore. This will be implemented in the master as well after the bottom nav bar is backported.

There is a risk for the `MainViewModel` to become a god class that handles everything; we can deal with it later after enough logic is moved here from fragments/activities and it becomes clear how to break it up. Right now it's still not very large.